packages/build-utils/src/file-fs-ref.ts

```diff
@@ -5,7 +5,9 @@ import path from 'path';
 import Sema from 'async-sema';
 import { FileBase } from './types';
 
-const semaToPreventEMFILE = new Sema(20);
+const semaToPreventEMFILE = new Sema(
+  parseInt(process.env.VERCEL_INTERNAL_FILE_FS_REF_SEMA, 10) || 20
+);
```

Comment:
```suggestion
const semaToPreventEMFILE = new Sema(
  parseInt(process.env.VERCEL_INTERNAL_FILE_FS_REF_SEMA || '20')
);
```

Reply:
Not sure why TS was yelling but the second arg to `parseInt` is the radix and it's a best practice to always use it to prevent unexpected behavior for some inputs. The result should be a number so I'll check the error in detail in the morning.

Reply:
The error is that the env var can be undefined and it doesnt like that

Reply:
yup. fixed